examples/bootstrap/index.html

```diff
@@ -53,10 +53,6 @@
               <a class="nav-link disabled" href="#">Disabled</a>
             </li>
           </ul>
-          <form class="form-inline mt-2 mt-md-0">
```

Comment:
Removed the search box because we dont want to confuse anyone who might expect actual working search (coming soon to dev portal, not api component).